com.syntevo.SmartGit.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.syntevo.SmartGit</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <name>SmartGit</name>
+  <summary>Get your commit done</summary>
+  <description>
+    <p>
+      SmartGit is a graphical Git and Mercurial client which can also connect to SVN repositories. SmartGit runs on
+      Linux, Mac OS X (10.11 or newer) and Windows (7 or newer) with the same intuitive user interface.
+    </p>
+    <p>
+      SmartGit comes with special integrations for GitHub, BitBucket and BitBucket Server (former Atlassian Stash) to
+      create and resolve Pull Requests and Review Comments. Of course, you can use SmartGit like any other Git client
+      with your own Git repositories or other hosting providers (e.g. GitLab).
+    </p>
+    <p>
+      Find out more on <a href="https://www.syntevo.com/smartgit/purchase/" target="_blank">https://www.syntevo.com/smartgit/purchase/</a>
```

Comment:
This isn't actually HTML, I suspect your `target` usage won't work. At the the very least confirm you tested this in GNOME-Software.